concepts/packages/introduction.md

```diff
@@ -1,52 +1,27 @@
 # Introduction
 
-TODO: the content below is copied from the exercise introduction and probably needs rewriting to a proper concept introduction
-
-## packages
-
-## functions
-
-## variables
-
-In Go an application is organized in packages. A package is a collection of source files located in the same folder. All source files in a folder must have the same package name at the top of the file. The package name is preferred to be the same as the folder it is located in.
+In Go an application is organized in packages. A package is a collection of source files located in the same folder. All source files in a folder must have the same package name at the top of the file. The package is preferred to be the same as the folder it is located in.
```

Comment:
Can we break this up into one line per sentence so that the blast radius for later changes is reduced?

```
In Go an application...
A package is...
All source files...
The package is preferred...
```

I think this was copy-pasted so could you fix the source as well?